pwndbg/dbg/__init__.py

```diff
@@ -172,6 +172,15 @@ def address(self) -> Value | None:
         """
         raise NotImplementedError()
 
+    # is_optimized_out is kind of a janky piece of API, honestly. It makes it
+    # so that one's ability to call all other methods in this class is often
+    # conditional on it being false, and it effectively splits the type into
+    # two.
+    #
+    # There's only _one_ part of Pwndbg that uses it, and I really feel like we
+    # should handle variables that have been optimized out some other way.
+    #
+    # TODO: Remove uses of is_optimized_out from plist and get rid of this.
```

Comment:
Do we know that other way we are going to handle it?

Reply:
I personally feel like returning some other type is a little more elegant here. Something like

```python3
class OptimizedOutValue:
    pass
    
OPTIMIZED_OUT = OptimizedOutValue()

# ...

if value == OPTIMIZED_OUT:
    # etc.
```

But that might just be my being used to Rust making me feel like this is clearer in lieu of using a Rust-style enum, which Python doesn't have.